app/furniture/marketplace/marketplace.rb

```diff
@@ -91,5 +94,17 @@ def create_stripe_webhook_endpoint(webhook_url:)
     def self.model_name
       @_model_name ||= ActiveModel::Name.new(self, ::Marketplace)
     end
+
+    def square_environment
+      if Rails.env.production?
+        "production"
+      end
```

Comment:
I would prefer if this was a value stored in the `Marketplace`; since we do need to be able to use the sandbox environment in a Rails production context.

Even if we don't make it to the UI; it does not seem appropriate to flag based upon deployment environment.

Reply:
I may have misinterpreted your earlier feedback as _put on marketplace model_ for some reason. Oof, tough getting used to jumping in and out of a codebase on limited time!

Reply:
Well, this is definitely the right *place* for it in our current domain model; it's just not quite how I would decide whether it's prod or not.

And yea, the "put work down and  pick it back up in narrow time windows" is a hard skill!